src/canvastools/ts/CanvasTools/Core/RegionData.ts

```diff
@@ -145,6 +145,39 @@ export class RegionData implements IRegionData, IMovable, IResizable {
         return area;
     }
 
+    public getLineSegments(): Array<[Point2D, Point2D]> {
```

Comment:
Tests coming in next push?

Reply:
I'll look into the test situation, there don't seem to be any in the application right now so the test framework may need to be setup or fixed up.

Reply:
Yes, the lint werror thing, and this lack of tests especially, I think we should definitely create some tasks to pick up later for fixing.